compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeParamAnalyzer.java

```diff
@@ -273,6 +273,10 @@ private BType createBuiltInType(BType type, Name name, long flags) {
                 anydataType.name = name;
                 anydataType.flags |= flags;
                 return anydataType;
+            case TypeTags.MAP:
```

Comment:
Not sure if we can consider map as a built-in type in this context based on the other types we are handling here. Do we know where it breaks if we do not add this here?

Reply:
yes. we can remove it. i ll remove it